Modules/Quotas/src/Quotas.Infrastructure/Persistence/Database/EntityConfigurations/TierQuotaDefinitionEntityTypeConfiguration.cs

```diff
@@ -0,0 +1,16 @@
+﻿using Backbone.Modules.Quotas.Domain.Aggregates.Tiers;
+using Microsoft.EntityFrameworkCore;
+using Microsoft.EntityFrameworkCore.Metadata.Builders;
+
+namespace Backbone.Modules.Quotas.Infrastructure.Persistence.Database.EntityConfigurations;
+
+internal class TierQuotaDefinitionEntityTypeConfiguration : IEntityTypeConfiguration<TierQuotaDefinition>
+{
+    public void Configure(EntityTypeBuilder<TierQuotaDefinition> builder)
+    {
+        builder.HasKey(x => x.Id);
+        builder.Property(x => x.Period);
+        builder.Property(x => x.MetricKey);
+        builder.Property(x => x.Max);
```

Comment:
That's the same as what EF Core does by convention. So you can remove all of this.

Reply:
I had to manually add this, otherwise it would not work.

The lack of a HasKey call results in:
`The entity type 'TierQuotaDefinition' requires a primary key to be defined.`

The lack of the Property calls, results in the columns not existing in the migration:
```
protected override void Up(MigrationBuilder migrationBuilder)
{
    migrationBuilder.DropColumn(
        name: "Max",
        table: "TierQuotaDefinitions");

    migrationBuilder.DropColumn(
        name: "MetricKey",
        table: "TierQuotaDefinitions");

    migrationBuilder.DropColumn(
        name: "Period",
        table: "TierQuotaDefinitions");
}
```